DateSeer/ChangeDatabase.cs

```diff
@@ -38,33 +38,65 @@ public ChangeDatabase(string picturePath, User user)
             }
 
         }
-        public ChangeDatabase(int MainId, int Id)
+        public ChangeDatabase(int MainId, int Id,int number,string table, string collumname1, string collumname2)
```

Comment:
Two things here:
* Too many parameters, make it a class.
* name variables more descriptively, avoid `column1`, `column2` etc.
* What is the difference between `MainId` and `Id`? It wasn't quickly glancing at it